bin/src/command/mod.rs

```diff
@@ -71,34 +68,12 @@ enum CommandMessage {
         worker_id: u32,
     },
     Advancement {
-        request_identifier: RequestIdentifier,
+        client_id: String,
         response: Response,
     },
     MasterStop,
 }
 
-/// identifies a request only within the command server
-/// the request part does NOT get sent to a worker
-#[derive(PartialEq, Eq, Clone, Debug)]
-pub struct RequestIdentifier {
```

Comment:
Could you implements `Display` trait as well to get a nice output easily which is a join of `client` and `request` fields?